icon-hider@kalnitsky.org/metadata.json

```diff
@@ -1,5 +1,5 @@
 {
-    "shell-version": ["3.26", "3.28", "3.30"],
+    "shell-version": ["3.26", "3.28", "3.30", "3.32"],
```

Comment:
Your new revised extension does not work with gs <3.32, so the metadata should reflect that. We can have both the new and old version active on extensions.gnome.org, so that people get the one that works with their particular gs.

Reply:
Thanks, I've pushed an updated version declaring support for only 3.32. For future reference, is there a good method of checking compatibility (using TravisCI or something) of different GNOME versions? I have a feeling that a lot of extensions extend their lists of supported versions without pruning them.